Tests/NIOTests/ByteBufferTest.swift

```diff
@@ -1438,44 +1438,94 @@ class ByteBufferTest: XCTestCase {
-    testAllocationOfReallyBigByteBuffer_state = .freeDone
-    /* free the pointer initially produced by malloc and then rebased by realloc offsetting it back */
-    free(ptr?.advanced(by: Int(Int32.max)))
-}
-
-private func testAllocationOfReallyBigByteBuffer_mallocHook(_ size: Int) -> UnsafeMutableRawPointer? {
-    precondition(AllocationExpectationState.begin == testAllocationOfReallyBigByteBuffer_state)
-    testAllocationOfReallyBigByteBuffer_state = .mallocDone
-    /* return a 16 byte pointer here, good enough to write an integer in there */
-    return malloc(16)
-}
-
-private func testAllocationOfReallyBigByteBuffer_reallocHook(_ ptr: UnsafeMutableRawPointer?, _ count: Int) -> UnsafeMutableRawPointer? {
-    precondition(AllocationExpectationState.mallocDone == testAllocationOfReallyBigByteBuffer_state)
-    testAllocationOfReallyBigByteBuffer_state = .reallocDone
-    /* rebase this pointer by -Int32.max so that the byte copy extending the ByteBuffer below will land at actual index 0 into this buffer ;) */
-    return ptr!.advanced(by: -Int(Int32.max))
-}
-
-private func testAllocationOfReallyBigByteBuffer_memcpyHook(_ dst: UnsafeMutableRawPointer, _ src: UnsafeRawPointer, _ count: Int) -> Void {
-    /* not actually doing any copies */
+    
+    private var testAllocationOfReallyBigByteBuffer_state = AllocationExpectationState.begin
+    private func testAllocationOfReallyBigByteBuffer_freeHook(_ ptr: UnsafeMutableRawPointer?) -> Void {
+        precondition(AllocationExpectationState.reallocDone == testAllocationOfReallyBigByteBuffer_state)
+        testAllocationOfReallyBigByteBuffer_state = .freeDone
+        /* free the pointer initially produced by malloc and then rebased by realloc offsetting it back */
+        free(ptr?.advanced(by: Int(Int32.max)))
+    }
+    
+    private func testAllocationOfReallyBigByteBuffer_mallocHook(_ size: Int) -> UnsafeMutableRawPointer? {
+        precondition(AllocationExpectationState.begin == testAllocationOfReallyBigByteBuffer_state)
+        testAllocationOfReallyBigByteBuffer_state = .mallocDone
+        /* return a 16 byte pointer here, good enough to write an integer in there */
+        return malloc(16)
+    }
+    
+    private func testAllocationOfReallyBigByteBuffer_reallocHook(_ ptr: UnsafeMutableRawPointer?, _ count: Int) -> UnsafeMutableRawPointer? {
+        precondition(AllocationExpectationState.mallocDone == testAllocationOfReallyBigByteBuffer_state)
+        testAllocationOfReallyBigByteBuffer_state = .reallocDone
+        /* rebase this pointer by -Int32.max so that the byte copy extending the ByteBuffer below will land at actual index 0 into this buffer ;) */
+        return ptr!.advanced(by: -Int(Int32.max))
+    }
+    
+    private func testAllocationOfReallyBigByteBuffer_memcpyHook(_ dst: UnsafeMutableRawPointer, _ src: UnsafeRawPointer, _ count: Int) -> Void {
+        /* not actually doing any copies */
+    }
+    
```

Comment:
Every change above here should be removed: all the code is the same, it's just been reflowed and confuses the diff.